### ISSUE TITLE
most diacritics feel slightly too light

### ISSUE THREAD

**Issue by EbenSorkin** [googlefonts/MetrophobicFont#2]

**Comment by aoifemooney** [comment]
yes, there were lots of inconsistencies there, i had fixed these previously also, hopefully they are better now — top new, bottom old.

<img width="896" alt="screen shot 2018-02-25 at 16 57 48" src="https://user-images.githubusercontent.com/12103183/36647000-1f08edda-1a4d-11e8-8cf4-778bd8a97d14.png">
